src/components/transactions/TxList/index.tsx

```diff
@@ -15,8 +18,24 @@ export const TxListGrid = ({ children }: { children: ReactNode }): ReactElement
   return <Box className={css.container}>{children}</Box>
 }
 
+const adjustPendingLabel = (items: TransactionListPage['results']) => {
+  const result = [...items]
+
+  if (isLabelListItem(result[0])) {
+    result[0] = {
+      label: 'Pending' as LabelValue,
```

Comment:
Why is it in two places now?

Reply:
We have it as part of `usePendingTxsQueue` which is only visible for 1/n safes. I didn't want to hide/filter items from the normal queue so instead I am adjusting the label there and hide items from this untrusted queue thats only visible for 1/n safes.

Reply:
As discussed lets not change this label and keep it as "Next" since this is the same behaviour in n/m safes anyway.